chromadb/test/test_api.py

```diff
@@ -683,7 +769,94 @@ def test_delete_where_document(api_fixture, request):
     collection.delete(where_document={"$contains": "great"})
     assert collection.count() == 0
 
-# endregion 
 
+logical_operator_records = {
```

Comment:
going to add a test for `AND` and `OR` used at the same time

Reply:
thank you!